.github/workflows/push.yml

```diff
@@ -21,6 +21,7 @@ jobs:
         DOCKER_REGISTRY_URL: ${{ secrets.DOCKER_REGISTRY_URL }}
         DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
         DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+        GO111MODULE: "on"
```

Comment:
This needs to be set in the `operator-sdk` step instead I believe.

Reply:
Um, yeah, that.